readthedocs/settings/docker_compose.py

```diff
@@ -220,6 +220,7 @@ def DATABASES(self):  # noqa
     # Remove the checks on the number of fields being submitted
     # This limit is mostly hit on large forms in the Django admin
     DATA_UPLOAD_MAX_NUMBER_FIELDS = None
+    SUPPORT_EMAIL = "example@company.com"
```

Comment:
```suggestion
    SUPPORT_EMAIL = "support@example.com"
```

Looks like company.com is valid, example.com is reserved for testing.